src/main/java/Task/Task.java

```diff
@@ -0,0 +1,82 @@
+package Task;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Abstract class representing a task with a completion status and formatting options.
+ * Hello!
+ */
+public abstract class Task {
+    protected static final DateTimeFormatter toSelfFormatter = DateTimeFormatter.ofPattern("MM-dd-yyyy HHmm");
+    protected static final DateTimeFormatter toUserFormatter = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
+
+    private final String task;
+    public String type;
+    private boolean completed;
```

Comment:
since this is a boolean variable, it would be better to name it with prefixes - in this case, perhaps `isCompleted`

Reply:
just noted your method is name `isCompleted`, perhaps that can be replaced with `hasBeenCompleted`?